css/main.css

```diff
@@ -400,3 +401,115 @@ button {
   top: 56px;
   gap: 8px;
 }
+
+/* styling form section */
+
+.form-section {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
```

Comment:
Your form section is currently displaying a row. As a result, your page structure is not corresponding to figma design on all devices.

Figma design:
![image](https://user-images.githubusercontent.com/42799579/192536213-fe7e32d1-0135-4fd0-a184-f02a6e9ed07a.png)

Your design:
![image](https://user-images.githubusercontent.com/42799579/192536356-92946a6c-35d8-4af2-894b-8b965a52b738.png)

TIP: To fix this, you can update your flex-direction to column 😀!